pages/solutions/reverse-etl.mdx

```diff
@@ -0,0 +1,180 @@
+---
+category: "Reverse ETL"
+pros:
+  [
+    "help sales and marketing teams understand their data",
+    "allow your data to move between sources with ease",
+  ]
+features:
+  [
+    {
+      title: "🔎 Use the profiles and segments you already have",
+      text: "Reverse ETLs let you use the robust profile data you've already made in your data warehouse.",
+    },
+    {
+      title: "💼  Make informed business decisions with operational analytics",
+      text:
+        "Reverse ETL tools help you make informed decisions that benefit your business.
+
+        ",
+    },
+    {
+      title: "📈  Improve advertising impact",
+      text: "With a Reverse ETL, you can automate list creation for effective advertising campaigns -- saving you advertising dollars and time.",
+    },
+    {
+      title: "🛍  Market more meaningfully",
+      text: "Help your marketing team create campaigns based on real, meaningful data.",
+    },
+  ]
+---
+
+## What is Reverse ETL and why is it important?
+
+Reverse ETL is a new data engineering term that has been picking up attention in recent months.
+
+At Grouparoo, we’re helping to lead that trend, so we thought we’d share an overview of what Reverse ETL is, and how Grouparoo enables businesses to use their data to the fullest extent.
+
+### What is Reverse ETL?
+
+Before we discuss Reverse ETL, let’s talk about “regular” ETL.
+
+ETL is a data warehouse term which stands for Extract, Transform, and Load. ETL describes the process of taking data from a database or source (Extract), modifying it or cleaning it up (Transform), and then putting it in a data warehouse (Load).
+
+<SectionDivider
+  imagePath="/images/home/solutions/categories/etl.svg"
```

Comment:
Not a blocker, but one of the more important aspects of ETL is that it's pulling data from a product database as well as data from SaaS tools like Salesforce. I'd suggest we replace one of the logos with a database logo or icon, but looks good overall!

Reply:
Sounds good.  I'll pop that into the tracker.